docs/content/authorization-concepts.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Authorization Concepts
+description: Introduction to Authorization 
+sidebar_position: 1
+slug: /authorization-concepts
+---
+
+import { ProductName, ProductNameFormat, RelatedSection } from '@components/Docs';
+
+# Authorization Concepts 
+
+## Authentication vs Authorization
+
```

Comment:
Add . here -- If we do decide to keep, ref modern TW practices:

Don't use. Instead, use more precise terms like persistent versus dynamic or manually configured versus automated. For more information, see [Avoid unnecessarily violent language](https://developers.google.com/style/inclusive-documentation#violent-language).